subcommand/server-acl-init/command.go

```diff
@@ -491,6 +497,60 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if len(c.flagIngressGateways) > 0 {
+		// Create a token for each ingress gateway name
+		for _, gateway := range c.flagIngressGateways {
+			// Check that the gateway name is not an empty string
+			if gateway == "" {
+				c.log.Error("Ingress gateway names cannot be empty")
+				return 1
+			}
+
+			// Define the gateway rules
+			ingressGatewayRules, err := c.ingressGatewayRules(gateway)
+			if err != nil {
+				c.log.Error("Error templating ingress gateway rules", "gateway name", gateway, "err", err)
+			}
+
+			// Names can be user specified in the Helm chart, and so don't
+			// append `-ingress-gateway`. For the ACL components, however, it
+			// is helpful to know the type of component.
+			name := fmt.Sprintf("%s-ingress-gateway", gateway)
```

Comment:
So for more folks this is going to end up with `ingress-gateway-ingress-gateway` and `terminating-gateway-terminating-gateway`. What do you think about suffixing with `-igw` and `-tgw` instead?